src/pycram/description.py

```diff
@@ -219,6 +220,53 @@ def __init__(self, _id: int, link_description: LinkDescription, obj: Object):
         self._current_pose: Optional[Pose] = None
         self.update_pose()
 
+    def get_bounding_box(self, rotated: bool = False) -> Union[AxisAlignedBoundingBox, RotatedBoundingBox]:
+        """
+        :param rotated: If True, return the rotated bounding box, otherwise the axis-aligned bounding box.
+        :return: The axis-aligned or rotated bounding box of a link. First try to get it from the simulator, if not,
+         then calculate it depending on the type of the link geometry.
```

Comment:
Add return to doc string

Reply:
What do you mean? there is already a :return: in the doc string.

Reply:
Good question